src/renderer/components/common/QrCode/QrReader/QrReader.tsx

```diff
@@ -128,9 +130,9 @@ const QrReader = ({
 
     if (fountainResult) {
       // decode the 1st frame --> it's a single frame QR
-      const result = EXPORT_ADDRESS.decode(fountainResult.slice(3));
+      let result = EXPORT_ADDRESS.decode(fountainResult.slice(3));
```

Comment:
Why do we need let here?

Reply:
this is made for 1st and next frames handling. In the 1st frame the app is able to understand which kind of QR is present: with the string or Raptor encoded data structure.
In the 2nd and next frames handler the application parses the QR code and accumulates Raptor frames.